types/sanctuary/index.d.ts

```diff
@@ -12,7 +12,11 @@ export as namespace S;
 
 type Nullable<A> = A | null;
 
-type Pair<A, B> = [A, B];
+interface Pair<A, B> {
+  constructor: {
+    '@@type': 'sanctuary/Pair';
+  };
+}
```

Comment:
The `@@type` property now lives on the values themselves:

```javascript
> (require ('sanctuary/package.json')).version
'3.0.0'

> var S = require ('sanctuary')
undefined

> (S.Pair (0) (0)).constructor['@@type']
undefined

> (S.Pair (0) (0))['@@type']
'sanctuary-pair/Pair@1'
```

```suggestion
interface Pair<A, B> {
  '@@type': 'sanctuary/Pair';
}
```

Please update the `Maybe<A>` and `Either<A, B>` interfaces as well. :)

Reply:
ah, cool! will do 👍